omod-2.0/src/main/java/org/openmrs/module/webservices/rest/web/v1_0/wrapper/AdministrationSectionLinks2_0.java

```diff
@@ -0,0 +1,59 @@
+/**
+ * This Source Code Form is subject to the terms of the Mozilla Public License,
+ * v. 2.0. If a copy of the MPL was not distributed with this file, You can
+ * obtain one at http://mozilla.org/MPL/2.0/. OpenMRS is also distributed under
+ * the terms of the Healthcare Disclaimer located at http://openmrs.org/license.
+ *
+ * Copyright (C) OpenMRS Inc. OpenMRS is a registered trademark and the OpenMRS
+ * graphic logo is a trademark of OpenMRS Inc.
+ */
+package org.openmrs.module.webservices.rest.web.v1_0.wrapper;
+
+import org.openmrs.module.web.extension.AdministrationSectionExt;
+
+import java.util.Map;
+
+/**
+ * REST wrapper for {@link AdministrationSectionExt}
```

Comment:
We also need a detailed description explaining why we are using this wrapper.

Reply:
Added here: https://github.com/openmrs/openmrs-module-webservices.rest/pull/484/commits/cce1f2d4bd949380e38e84d3b161bb1bf4197866